app/forms/Choice.scala

```diff
@@ -16,22 +16,28 @@
 
 package forms
 
-import play.api.data.Form
-import play.api.data.Forms.{mapping, text}
+import play.api.data.Forms.{optional, text}
+import play.api.data.{Form, Forms, Mapping}
 import play.api.libs.json.Json
+import utils.validators.forms.FieldValidator.isContainedIn
 
 case class Choice(choice: String)
 
 object Choice {
   implicit val format = Json.format[Choice]
 
+  val choiceId = "Choice"
+
   import AllowedChoiceValues._
   private val correctChoice = Set(SupplementaryDec, StandardDec, Arrival, Departure, CancelDec)
```

Comment:
If you can please change name to `correctChoices`

Reply:
Yes, I can.